include/ibamr/IIMethod.h

```diff
@@ -168,6 +168,14 @@ class IIMethod : public IBStrategy
         void* ctx;
     };
 
+    /*!
+     * Returns either the current configuration or the
+     * reference configuration of the interface at the specified time.
+     *
+     */
+    libMesh::NumericVector<double>*
+    getMeshCoordinates(bool is_current_configuration, std::string time, unsigned int part);
```

Comment:
```suggestion
    getInteractionConfiguration(std::string time, unsigned int part);
```
Let's simplify this interface; the implementation should be able to determine what configuration to use based on the internal class state.

Reply:
I would rather just pass in the time when the configuration is needed; the implementation can figure out which vector to use based on the current and new times.